docs/Cloud_integrations/kafka.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Kafka
+date: 2023-08-08
+og_title: AVSystem IoT Developer Zone
+---
+
+# Kafka
+
+Connect {{ coiote_long_name }} to your Kafka cluster by creating a Kafka event handler.
```

Comment:
I think it's worth pointing out that you can connect with `Azure Event Hub`, e.g. by writing `to your Kafka cluster or Azure Event Hub` - I think more people are used to working with Azure services than to working with Kafka clusters (either managed by themselves or by other companies)